visualizer/index.html

```diff
@@ -467,6 +470,7 @@
 
   <div id="options">
     <ul>
+      <li><input type="checkbox" name="showExampleGenerator"><label>Show Example Generator</label></li>
```

Comment:
I meant for the label. Ok, feel free to leave it as "Show Example Generator" but please change to sentence case to be consistent.